isso/js/app/i18n.js

```diff
@@ -101,22 +93,46 @@ define(["app/config", "app/i18n/bg", "app/i18n/cs", "app/i18n/da",
         oc: oc,
         pl: pl,
         pt: pt_BR,
-        pt_BR: pt_BR,
+        "pt-BR": pt_BR,
         ru: ru,
         sk: sk,
         sv: sv,
         nl: nl,
         vi: vi,
         zh: zh_CN,
-        zh_CN: zh_CN,
-        zh_TW: zh_TW
+        "zh-CN": zh_CN,
+        "zh-TW": zh_TW
     };
 
-    var plural = pluralforms(lang);
+    // for each entry in config.langs, see whether we have a catalogue
+    // entry and a pluralforms entry for it.  if we don't, try chopping
+    // off everything but the primary language subtag, before moving
+    // on to the next one.
+    var lang, plural, translations;
+    for (var i = 0; i < config.langs.length; i++) {
+        plural = pluralforms(lang);
```

Comment:
This fails for me since `lang` is none.
```
Uncaught TypeError: lang is undefined
```
Did you mean to add `lang = config.langs[i];` before attempting to call `pluralforms(lang)`?

Reply:
Doh, yes I did.  I do most of my programming in Python so I'm used to for-loops over arrays giving me the value, not the index.  Thanks for catching, will be fixed in the next push.